pkg/queue/manager.go

```diff
@@ -199,6 +201,7 @@ func (m *Manager) UpdateQueue(q *kueue.Queue) error {
 		}
 	}
 	qImpl.update(q)
+	qImpl.reportPendingWorkloads()
```

Comment:
How about wrap with `defer`, difference here is when return `errQueueDoesNotExist` error, we will still run `reportPendingWorkloads()`. (Similar to other places)